Python/Product/PythonTools/PythonTools/Project/ImportWizard/ImportSettings.cs

```diff
@@ -454,7 +454,6 @@ InterpreterConfiguration config
                 new Dictionary<string, string> {
                     { MSBuildConstants.IdKey, Path.GetFileName(sourcePath) },
                     { MSBuildConstants.DescriptionKey, config.Description },
-                    { MSBuildConstants.BaseInterpreterKey, config.Id },
```

Comment:
Please confirm this is correct. This is based on what I see in .pyproj when I add a new virtual environment to an existing project.